tests/benchmarks/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@
 
 add_definitions(-DCASBIN_PROJECT_DIR=${CMAKE_SOURCE_DIR})
 
+option(INTENSIVE_BENCHMARK OFF)
```

Comment:
Good job. Maybe adding a commet like [that](https://github.com/casbin/casbin-cpp/blob/abaa2174f0a4c6df2a25c21c388c15939f472655/CMakeLists.txt#L54) is better.

Reply:
I followed your suggestion, please review again. Thanks.